tool/jt.rb

```diff
@@ -522,11 +537,11 @@ def build(*options)
       require 'etc'
       cores = Etc.respond_to?(:nprocessors) ? Etc.nprocessors : 4
       raw_sh "make", "-j#{cores}", chdir: "#{TRUFFLERUBY_DIR}/src/main/c"
+    when 'native'
+      build_truffleruby
+      build_native_image
```

Comment:
`build_native_image` already calls `build`, so I guess the line above can be dropped or, the line in build_native_image.

Reply:
Oops. I thought that was calling the graal build. I'll fix that.